.github/workflows/combine.yml

```diff
@@ -30,6 +30,14 @@ on:
     # Rebuild on Ruby infrastructure changes.
     - 'Gemfile'
 
+    # Dependencies
+    - 'FirebaseCore/**'
+    - 'FirebaseAuth/**'
+    - 'FirebaseFunctions/**'
+    - 'Firestore/**'
+    - 'FirebaseStorage/**'
+    - 'FirebaseStorageSwift/**'
```

Comment:
WDYT about adding the deps of `FirebaseCombineSwift` to catch these breaks on PRs in the future?

Reply:
SGTM, maybe sufficient to do 'FirebaseCore/**/*.h', etc though